sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 import { getUniqueName, isBrowser, isRecordMode, isPlaybackMode } from "./utils";
-import { NiseRecorder, NockRecorder, BaseRecorder, setEnviromentOnLoad } from "./baseRecorder";
+import { NiseRecorder, NockRecorder, BaseRecorder, setEnvironmentOnLoad } from "./baseRecorder";
```

Comment:
Please update line 75 as well. Running rushx build locally should show where the TS errors are coming from :)

Reply:
gosh darnit, I'm clumsy today. Thank you Christopher!